### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zeibura**
Signed-off-by: Zeibura Kathau <zeibura.kathau@broadcom.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ With the Code4z extension pack you can:
...
+* 2021-05-24 - [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe) version 1.15.1, see [Changelog](https://marketplace.visualstudio.com/items/Zowe.vscode-extension-for-zowe/changelog).
 * 2021-04-08 - [Debugger for Mainframe](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.debugger-for-mainframe) version 1.5, see [Changelog](https://marketplace.visualstudio.com/items/broadcomMFD.debugger-for-mainframe/changelog).
-* 2021-03-26 - [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe) version 1.13.1, see [Changelog](https://marketplace.visualstudio.com/items/Zowe.vscode-extension-for-zowe/changelog).
 * 2021-03-15 - [COBOL Langage Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) version 0.18.0, see [Changelog](https://marketplace.visualstudio.com/items/broadcomMFD.cobol-language-support/changelog).
```
Comment (zimlu02) [inline README.md:30]: as agreed, let's wait for the HLASM release, that will come early next week.

**Review by zimlu02 — CHANGES_REQUESTED** [review]

**Review by jellypuno — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -81,30 +82,42 @@ You can learn more about the Zowe Explorer by watching the [Getting Started](htt
...
+The Explorer for Endevor extension modernizes the way you interact with [CA Endevor]((https://www.broadcom.com/products/mainframe/devops-app-development/app/endevor-software-change-manager)), offering a user-friendly and convenient way to work with elements and inventory locations.
 
 This extension offers best developer experience in synergy with [Bridge for Git](https://youtu.be/sjnZuQpUVM4), a solution which enables you to concurrently work in Git and mainframe.
 
 > How can we improve Explorer for Endevor? [Let us know on our Git repository](https://github.com/eclipse/che-che4z-explorer-for-endevor/issues)
 
 ### Features
-* Retrieve, browse and search CA Endevor® elements.
+
+- View an element
```
[the remark this answers is not in the document]
Comment (IgorCATech) [reply README.md:113]: yes, that's a good suggestion actually. Let me go ahead and change it

File: README.md (hunk 1 above)
Comment (zeibura) [reply README.md:30]: @zimlu02, Dejan was suggesting that we add info on IDMS support to high-level descriptions of COBOL LS. What do you think of this? Would this PR be the right time to do it, or should we wait for the next COBOL release?

Comment (zeibura) [reply README.md:31]: nevermind - we will do this when we update Code4z next time after the COBOL release.

**Review by zimlu02 — APPROVED** [review]

**Comment by zeibura** [comment]
please merge https://github.com/BroadcomMFD/code4z/pull/32 before merging this PR

**Review by zimlu02 — CHANGES_REQUESTED** [review]
- please update also COBOL LS to 0.19.0 (will be released today)
- add CCF into What's new section, highlighting that this is newly added to the package

**Review by zimlu02 — APPROVED** [review]

**Pull request merged**